src/scripts/components/elements/validation.js

```diff
@@ -66,4 +73,49 @@ export default class Validation {
       'h5p-reader-question-required-wrapper', 'h5p-reader-question-hidden'
     );
   }
+
+  /**
+   * Validate input/editor content
+   * @param {number} contentLength length of content
+   * @returns {boolean}.
+   */
+  validate(contentLength) {
+    const remainingChars = this.params.charactersLimit - contentLength;
+    this.params.fieldContainer.classList.remove('validation-error');
+    let isError = this.validateCharsLimit(contentLength) ? true : false;
+
+    if (this.params.isRequired && contentLength === 0) {
+      this.params.fieldContainer.classList.add('validation-error');
+      this.showError(this.params.i10n.requiredMessage);
+      isError = true;
+    }
+
+    return isError;

```

Comment:
Hmm... I am not really following H5P rules for the validation but not sure if that is necessary in this case as well.

For simplicity and to avoid confusion I will rename the function.

Reply:
It's not necessary for us, no. But I am trying to follow the premise that even less experienced people should not have to struggle with H5P code too much. If `validate` always follows the same scheme in every corner of H5P, that's probably better - I know, just a tiny drop in the ocean, but still a drop.

I hope my comments do not come across as "This must be the way!!!".